docs/source/deployment/deploy_shm.md

```diff
@@ -54,7 +54,7 @@ Before deploying the Safe Haven Management (SHM) infrastructure you need to deci
   Use the short name without spaces, _e.g._ **uksouth** not **UK South**
   :::
 
-Once you've decided on these, run the following command: [approx 5 minutes]:
+Once you've decided on these, run the {typer}`dsh-shm-deploy` command: [approx 5 minutes]:
```

Comment:
If this is supposed to produce a link configured by `sphinxcontrib.typer` it isn't currently working for me (in the online preview).

Reply:
works locally, that's because the docs aren't generating correctly, as the module itself isn't installed on the ReadTheDocs runner so it can't just `import data_safe_haven`

Reply:
I suppose that is because it isn't part of the `requirements.txt` file.

[extra requirements](https://docs.readthedocs.io/en/stable/config-file/v2.html#python) should fix this I think.